gateway/src/main/java/org/georchestra/gateway/security/ldap/accounts/GeorchestraLdapAccessConfiguration.java

```diff
@@ -0,0 +1,123 @@
+package org.georchestra.gateway.security.ldap.accounts;
+
+import static java.util.Objects.requireNonNull;
+
+import org.georchestra.ds.orgs.OrgsDao;
+import org.georchestra.ds.orgs.OrgsDaoImpl;
+import org.georchestra.ds.roles.RoleDao;
+import org.georchestra.ds.roles.RoleDaoImpl;
+import org.georchestra.ds.roles.RoleProtected;
+import org.georchestra.ds.users.AccountDao;
+import org.georchestra.ds.users.AccountDaoImpl;
+import org.georchestra.gateway.security.ldap.LdapConfigProperties;
+import org.georchestra.gateway.security.ldap.extended.ExtendedLdapConfig;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.ldap.core.LdapTemplate;
+import org.springframework.ldap.core.support.LdapContextSource;
+import org.springframework.ldap.pool.factory.PoolingContextSource;
+import org.springframework.ldap.pool.validation.DefaultDirContextValidator;
+
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(LdapConfigProperties.class)
+public class GeorchestraLdapAccessConfiguration {
+
+    /**
```

Comment:
Hello, I would expect " @ConditionalOnProperty" here to check if ldap config props are there or not  ? is n' it ?

Reply:
Done with c10349d